flake.nix

```diff
@@ -181,7 +181,7 @@
           cargo-deny
           cargo-insta
           cargo-nextest
-          cargo-watch
+          bacon
```

Comment:
Nit: these look like they may have been alphabetized, please consider preserving the order if so.